db/payment_icons.yml

```diff
@@ -285,8 +285,12 @@
 -
   name: swish
   label: Swish
-  group: other
+  group: bank_transfers
 -
+  name: ratepay
+  label: RatePay
+  group: wallets  
+ -
```

Comment:
I believe this is the CI issue. This `-` is indented one space. We need to remove this indentation.

Reply:
@larouxn Thank you. Removed the space. I hope this will work now. Please review and suggest.